src/components/graph/PortfolioGraph.tsx

```diff
@@ -151,7 +155,43 @@ function makeRequest(reqUrl: string) {
   });
 }
 
-export default function PortfolioGraph() {
+function calculateNextDate(currentDate: Date, activeButton: number) : Date {
+  switch (activeButton) {
+    case 0:
+      return addHours(currentDate, 1);
+    case 1:
+      return addDays(currentDate, 1);
+    case 2:
+      return addDays(currentDate, 5);
+    case 3:
+      return addDays(currentDate, 15);
+    case 4:
+      return addMonths(currentDate, 2);
+    default:
+      return addMonths(currentDate, 12);
+  }
+}
+
+function generateEmptyData(fromDate: Date, toDate: Date, activeButton: number) {
```

Comment:
In the future if you add code like this, please make sure to add a //TODO comment so that we can keep track of what needs to be removed later on

Reply:
So this is essentially generating a flat-line graph?

Reply:
Yes, but with enough data for it to display ticks.